cli/determined_cli/checkpoint.py

```diff
@@ -55,61 +54,29 @@ def render_checkpoint(checkpoint: experimental.Checkpoint, path: str) -> None:
 
 @user.authentication_required
 def list(args: Namespace) -> None:
```

Comment:
Should we keep this function here or move it to `cli/experiments`? It's invoked there and no where else.